documentation/settings/topics/integrations.rst

```diff
@@ -5,13 +5,13 @@
 Integrations
 ************
 
-Hyperview comes with a few standard integrations that are built into either the main platform or the Data Collector.
+Hyperview has a few standard integrations built into the main platform or the Data Collector.
 
 - GoogleMaps APIs for address lookups and map-related visualization
 - AccuWeather APIs for weather forecast and alarms
 - VMware vSphere APIs for virtual machine discovery
 
-Hyperview APIs are documented from within the product and on the Hyperview Documentation site. Customers are free to write their integrations and there are API integration samples provided on `GitHub <https://github.com/HyperviewHQ/API-Samples>`_.
+Hyperview APIs are documented from within the product and on the Hyperview Documentation site. Customers can write their integrations, and API integration samples are provided on `GitHub <https://github.com/HyperviewHQ/API-Samples>`_.
```

Comment:
Grammar seems a bit odd. Perhaps:
Customers can write their own integrations

Or we can remove that completely, providing an open API implies they can write their own integration.

Reply:
Updated.